addons/spreadsheet/static/src/@types/global_filter.d.ts

```diff
@@ -3,9 +3,13 @@ import { DomainListRepr } from "@web/core/domain";
 
```

Comment:
Might need a scrollbar :upside_down_face: 

![image](https://github.com/user-attachments/assets/973ce2da-e699-4b0a-81ba-b4cf166f6620)

Reply:
https://www.odoo.com/odoo/project/2328/tasks/4875714 as I have to wait for frameworkjs input